spec/httparty/response_spec.rb

```diff
@@ -107,6 +107,11 @@
     expect(response.respond_to?(:parsed_response)).to be_truthy
   end
 
+  it "responds to predicates" do
+    response = HTTParty::Response.new(@request_object, @response_object, @parsed_response)
+    expect(response.respond_to?(:success?)).to be_truthy
```

Comment:
Kind of concerned that we'll unknowingly break compatibility since only success? is tested, but I guess we can roll and see what happens.